Formula/w/whisper-cpp.rb

```diff
@@ -33,11 +33,27 @@ def install
       -DWHISPER_BUILD_SERVER=OFF
     ]
     args << "-DLLAMA_METAL_MACOSX_VERSION_MIN=#{MacOS.version}" if OS.mac?
+    args << "-DCMAKE_INSTALL_RPATH=#{OS.mac? ? "@loader_path/../libinternal" : "'$ORIGIN/../libinternal'"}" if head?
```

Comment:
Use the `rpath` helper (and `build.head?`) instead:
```suggestion
    args << "-DCMAKE_INSTALL_RPATH=#{rpath(target: prefix/"libinternal")}" if build.head?
```